### PR TITLE
Query and Update

### PR DESCRIPTION
**Pull request by AJ193**
### Implementations in this project

- Use database transactions.
- Modify and delete data in SQL.
- Prepare complex queries that answer analytical questions.

- Add a column
step 1

- Insert the following data:
step 2

- Setting the species column to pokemon for all animals that don't have species already set.
step 3

- delete all records in the animals table, then roll back the transaction.
step 4

- Update all animals' weight to be their weight multiplied by -1
step 5

- Rollback to the savepoint
step 6

- Update all animals' weights that are negative to be their weight multiplied by -1.
step 7

- How many animals are there?
step 9

- How many animals have never tried to escape?
step 10

- What is the average weight of animals?
step 11

- Who escapes the most, neutered or not neutered animals?
step 12

- What is the minimum and maximum weight of each type of animal?
min-max weight

- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
step 15

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,61 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
...
+
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+
+SELECT AVG(weight_kg) FROM animals;
+
+SELECT MAX(escape_attempts), neutered FROM animals GROUP BY neutered;
+
+SELECT MIN(weight_kg) FROM animals;
+
+SELECT MAX(weight_kg) FROM animals;
```
Comment (BregornOriginal) [inline queries.sql]: - [x] While doing queries it's really useful to compare data and to get data. Kindly make these two queries in a single one to have a table with two values and can see easily the difference 🥇

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @AJ193  

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

giphy (5)

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good PR description

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (AJ193) [reply queries.sql]: @BregornOriginal Issue Resolved

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @AJ193, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**